src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -774,6 +775,24 @@ def clone_subvolume_snapshot(self, **kwargs):
             ret = self.volume_exception_to_retval(ve)
         return ret
 
+    def _get_clone_status(self, fsh, grph, svh, json_fmt=True):
```

Comment:
Shorter names are better than long names. But I've made the change you requesed - https://github.com/ceph/ceph/pull/54620/files#diff-adf1b497bf9b7b60e53497d8a88519f2e693b1fa9bd6854f9b72ac75944c61d1R789.

Reply:
> Shorter names are better than long names.

I disagree. Self explanatory names are better, however, super_long_self_explanatory_names are obviously not good too. There's a balance :)

Reply:
> > Shorter names are better than long names.
> 
> I disagree. Self explanatory names are better, however, super_long_self_explanatory_names are obviously not good too. There's a balance :)

True, balanced names are better than either of the extremes. We use short names like these in server, client and kernel code (like fs, fsp, smap, etc), so I thought this would be fine. No issues, I've incorporated the change you requested.